### PR TITLE
Update eslint config

### LINKED ISSUE #121: Add eslint rules for consistent imports
To have consistent imports, add eslint rules to encourage (and perhaps enforce?) a preferred ordering.

The rules in question:

https://eslint.org/docs/latest/rules/sort-imports

Stick with the default eslint sorting rules ([link](https://eslint.org/docs/latest/rules/sort-imports#options)), in case we are not happy with it, we can change it later.

Notes:
Should we reformat all files at once?

### PR DESCRIPTION
**Pull request by AbdBarho**
Continuation of #125 

Closes #121

Update eslint config & fix all linting errors

We warn instead of error for sort imports, since we can't just `eslint --fix`.

There are some other [tools](https://github.com/trivago/prettier-plugin-sort-imports) that can do this, we could integrate them later.

### CODE REVIEW

File: website/src/lib/prismadb.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { PrismaClient } from "@prisma/client";
 
 declare global {
-  var prisma;
+  const prisma;
```
Comment (yk) [inline website/src/lib/prismadb.ts:4]: the `const` here breaks build.

**Review by k-nearest-neighbor — APPROVED** [review]
LGTM and builds on latest

**Comment by k-nearest-neighbor** [comment]
I think you just need to rebase to fix the Header.tsx merge

**Comment by k-nearest-neighbor** [comment]
> I think you just need to rebase to fix the Header.tsx merge

NM, I resolved it through the github UI

**Pull request merged**